layouts/index.html

```diff
@@ -41,4 +41,31 @@ <h2>{{- .subtitle -}}</h2>
     </div>
 </div>
 {{- end -}}
+{{- with .Params.section4 -}}
+<div id="home-our-work" class="section">
+    <div class="container">
+        <div class="row header">
+            <div class="col-sm-8">
+                <h2>{{- .subtitle -}}</h2>
+            </div>
+            <div class="col-sm-4 end-sm">
+                <p>{{- .title -}}</p>
+            </div>
+        </div>
+        <!-- Passing casestudy content to the case study partial -->
+        {{- partial "site/casestudy" .casestudy -}}
+        {{- with .industries -}}
+        <div class="industries">
+            <h1>{{- .title -}}</h1>
+            <div class="row list">
+                {{- range .list -}}
+                <!-- Passing dict @name @info @url content to the link card partial -->
+                {{- partial "site/linkcard" (dict "name" .name "info" "" "url" "#") -}}
```

Comment:
Add `"context" .` as the first item in your dict. It's a good habit to take, and it makes sure you pass the general Hugo `.` to the partial (in case you need something like `.Site.BaseURL`).